neonvm/config/controller/deployment.yaml

```diff
@@ -59,6 +59,7 @@ spec:
         # * cache.direct=on    - use O_DIRECT (don't abuse host's page cache!)
         # * cache.no-flush=on  - ignores disk flush operations (not needed; our disks are ephemeral)
         - "--qemu-disk-cache-settings=cache.writeback=on,cache.direct=on,cache.no-flush=on"
+        - "--reconcile-failure-interval=10m"
```

Comment:
10 minutes is a very long time if we expect to reconcile every second. Maybe let's set this to 30s or 1m to start with?